test/units/modules/network/fortimanager/test_fmgr_provisioning.py

```diff
@@ -0,0 +1,63 @@
+# (c) 2016 Red Hat Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from nose.plugins.skip import SkipTest
```

Comment:
We'll also need to switch to [pytest](https://docs.pytest.org/en/latest/) for iunit tests as documented [here](https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#structuring-unit-tests). The Ansible Testing Community Group is attempting to standardize tests moving forward. Thank you!

Reply:
Please raise a follow up PR to fix this once this PR is merged